JournalCoreData/Model Controllers/EntryController.swift

```diff
@@ -113,7 +113,7 @@ class EntryController {
                 return
             }
 
-            let moc = CoreDataStack.shared.mainContext
+            let moc = CoreDataStack.shared.container.newBackgroundContext()
```

Comment:
This will cause the completion handler to be called on a background queue - consider calling the completion handler from the main queue to switch back